### DISCUSSION TITLE
Código de error: Out of Memory

### DISCUSSION THREAD

**Discussion by Agudelo02093** [sebasvelasco353/FDP-2022_01#42] [Q&A]
Hola profe :3
Estaba tratando de incluir las listas de opciones de países como nos lo mostraste en clase y al previsualizarlo me aparece este error, Código de error: Out of Memory. No sé si mi computador no da para esto o si tal vez puse algo mal que causa el error.

Te adjunto mis archivos para que puedas revisar. Muchas gracias.

![image](https://user-images.githubusercontent.com/98620510/153974537-27a103d7-4c41-4734-9de3-5d632c6d15ae.png)

![image](https://user-images.githubusercontent.com/98620510/153974709-c8c15469-17b7-4e67-a35b-8d9e40f3ed36.png)

Pdta: Te subo pantallazos porque no recuerdo como actualizar mi repositorio sin sobrescribir o eliminar lo que ya está :3... Si me lo recuerdas aprovechando este espacio te lo agradezco.

**Comment by sebasvelasco353** [comment]
No se me ocurre otra cosa diferente a lo de la memoria, intenta borrando elementos del arreglo y dejando unos 5 a ver que pasa.
dejame tambien porfa un screenshot del error y dejame tambien como estas abriendo el archivo (vscode live server o abriendo el archivo directamente con tu navegador) yo busco a ver que sera.
